examples/tutorials/self-paced-training/part-1_federated_learning_introduction/chapter-1_running_federated_learning_applications/01.4_client_side_customization/code/src/fl_job.py

```diff
@@ -18,7 +18,7 @@
 from src.fedavg import FedAvg
 from src.network import SimpleNetwork
 
-from nvflare.job_config.api import FedJob
+from nvflare.job_config import FedJob
```

Comment:
suggest change to from nvflare import FedJob, and suggest change this in another PR.